src/components/breadcrumb/Breadcrumb.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+    <nav :class="breadcrumbClasses">
+        <ul>
+            <template v-for="(item, index) in items">
+                <li
+                    v-if="item"
+                    :key="item"
+                    :class="{ 'is-active': index === items.length - 1 }">
+                    <router-link
```

Comment:
Buefy doesn't have any external dependency (except of Bulma and Vue) and then you should remove `router-link`. Please take a look at https://github.com/buefy/buefy/blob/dev/src/components/navbar/NavbarItem.vue